### ISSUE TITLE
[help please]cannot source a .sql file in snap

### ISSUE THREAD

**Issue by vanos001** [DownThePark/mysql-snap#4]
this is the error i get when it try to source a .sql file
```
mysql> source /home/<username>/Downloads/companydb.sql;
ERROR: 
Failed to open file '/home/<username>/Downloads/companydb.sql', error: 13
```
i tried to find the solution but cant seem to find any

**Comment by vanos001** [comment]
never mind found a way 
```mysql -u <user name> -p <db name> < smtg.sql```
should i close this issue then??

**Comment by vanos001** [comment]
thank you for fixing can now source the .sql file

**Comment by DownThePark** [comment]
Hey, @vanos001. Thanks for the bug report.

> thank you for fixing can now source the .sql file

Yup, this issue should now be fixed in commit [f0d3b4f](https://github.com/DownThePark/mysql-snap/commit/f0d3b4fd412fcf1591257aefcbe9a996ac1c4f50) for the `latest` track, and commit [b09862e](https://github.com/DownThePark/mysql-snap/commit/b09862e7bd207039673947289ed50065f9d36bf6) for the `8.0` track, both of which add support for the `home` interface.

Thanks again,
DownThePark
